buildtest/cli/__init__.py

```diff
@@ -1103,6 +1103,13 @@ def report_menu(subparsers, parent_parser):
     filter_group = parser_report.add_argument_group("filter", "Filter and Format table")
 
     # buildtest report
+    filter_group.add_argument(
+        "-d",
+        "--detailed",
+        action="store_true",
+        help="Print a detailed summary of the test results",
+    )
+
```

Comment:
can you put this argument under `optional arguments` and not under the `filter_group`  so when you run `--help` it should be where you see `--oldest`

```
optional arguments:
  -h, --help            show this help message and exit
  --pager               Enable PAGING when viewing result
  --row-count           Display number of rows from query shown in table
  -t, --terse           Print output in machine readable format
  -n, --no-header       Do not print header columns in terse output (--terse)
  -c COUNT, --count COUNT
                        Retrieve limited number of rows that get printed
  -f, --fail            Retrieve all FAIL tests
  -p, --pass            Retrieve all PASS tests
  -s START, --start START
                        Retrieve tests by starttime
  -e END, --end END     Retrieve tests by endtime
  --latest              Retrieve latest record of particular test
  --oldest              Retrieve oldest record of particular test

```